adc/ade7913/ade7913.c

```diff
@@ -247,10 +247,9 @@ int ade7913_sync(oid_t *device, const char *cs, int devcnt, int snap)
 	oid_t gpiodev;
 	platformctl_t pctl;
 
-	int muxes[4] = { pctl_mux_gpio_ad_18, pctl_mux_gpio_ad_19,
-		pctl_mux_gpio_ad_20, pctl_mux_gpio_ad_29 };
+	int muxes[4] = { pctl_mux_gpio_ad_29, pctl_mux_gpio_ad_18, pctl_mux_gpio_ad_19, pctl_mux_gpio_ad_20 };
 
-	int pins[4] = { 17, 18, 19, 28 };
+	int pins[4] = { 28, 17, 18, 19 };
```

Comment:
Perhaps take the opportunity to move this to board_config.h?

Reply:
there is too much hard-coded logic in irq handler `edma_spi_rcv_irq_handler` -  (raw access to GPIO port with the assumption all pins are on a single one, etc.) - I don't want to generalize it too much (adding multiple GPIO ports/for's, etc.) and risk the ISR will take longer than expected.